testonly/hammer/record.go

```diff
@@ -68,6 +68,7 @@ func (s *smrStash) pushSMR(smr types.MapRootV1) error {
 	return nil
 }
 
+// TODO(mhutchinson): consider making this return a bool or an error to indicate missing result.
```

Comment:
nit: select(`consider making this `) -> Trash.

Reply:
Agree.